data/category-porn

```diff
@@ -994,6 +1005,9 @@ bffshd.com
 biaoqs.xyz
 biaoqs1.xyz
 biaoqs2.xyz
+bibeidh16.cc
+bibeidh17.cc
+bibeidh18.cc
```

Comment:
Some domains you can try to use regular expression to match it.
`(\.|^)bibeidh1[6-8]\.cc$